tests/phpunit/api/v3/PaymentProcessorTypeTest.php

```diff
@@ -37,21 +35,29 @@ public function setUp() {
 
   /**
    * Check with no name.
+   * @dataProvider versionThreeAndFour
    */
-  public function testPaymentProcessorTypeCreateWithoutName() {
+  public function testPaymentProcessorTypeCreateWithoutName($version) {
+    $this->_apiversion = $version;
     $payProcParams = [
       'is_active' => 1,
     ];
     $result = $this->callAPIFailure('payment_processor_type', 'create', $payProcParams);
-    $this->assertEquals($result['error_message'],
-      'Mandatory key(s) missing from params array: name, title, class_name, billing_mode'
-    );
+    if ($version === 4) {
+      $errorMessage = 'Mandatory values missing from Api4 PaymentProcessorType::create: name, title, class_name, billing_mode';
+    }
+    else {
+      $errorMessage = 'Mandatory key(s) missing from params array: name, title, class_name, billing_mode';
+    }
```

Comment:
@seamuslee001 how about `$this->assertContains('name, title, class_name, billing_mode', $result['error_message']);`

Reply:
Done